packages/react-querybuilder/package.json

```diff
@@ -59,6 +59,7 @@
     "vite-tsconfig-paths": "^3.3.17"
   },
   "dependencies": {
+    "bson": "^4.6.4",
```

Comment:
Why was this package added? Where it's being used, it looks like you could just use `JSON.parse` instead of `EJSON.parse`. Even if `bson` was necessary, it should be in  `devDependencies`, not `dependencies`.

Reply:
For this PR it is not needed, i will remove it 👍🏻